Code/Editor/EditorEngineProcess/EngineProcGameApp.cpp

```diff
@@ -349,11 +349,7 @@ void ezEngineProcessGameApplication::EventHandlerIPC(const ezEngineProcessCommun
     if (pMsg->m_bDocumentOpen)
     {
       pDocumentContext = CreateDocumentContext(pMsg);
-
-      if (pDocumentContext == nullptr)
-      {
-        ezLog::Error("Could not create a document context for document type '{0}'", pMsg->m_sDocumentType);
-      }
+      // EZ_ASSERT_DEV(pDocumentContext != nullptr, "Could not create a document context for document type '{0}'", pMsg->m_sDocumentType);

```

Comment:
Why did you undo this ?

Reply:
Because some assets don't have a document context (all that don't have a preview) and for them it's totally fine. I couldn't find a good way to tell whether an asset needs a context or not.